src/routes/Start.svelte

```diff
@@ -20,43 +20,27 @@
 </section>
 {/if}
 
-<p>With this tool, you can produce a report of an authoring tool's conformance with the ATAG standard. The tool is designed for accessibility auditors and  requires a working knowledge of <a href="https://www.w3.org/TR/ATAG20/">ATAG 2.0</a>.</p>
+<p>Authoring tools include content management systems (CMS), code editors, and other software used to create content. With this tool, you can produce a report of an authoring tool's conformance with the <a href="https://www.w3.org/TR/ATAG20/">ATAG 2.0</a> standard. The tool is designed for accessibility auditors.</p>
 
-<details>
-	<summary><h2>What to expect</h2></summary>
-	<p>Following the structure of ATAG, this tool takes you through eight  <strong>Principles</strong>. Each principle has a number of <strong>Guidelines</strong>, which break down into <strong>Success Criteria</strong>. For each, you can select a result and note down any observations.</p>
-	<p>Part A (Principles A.1 to A.4) is related to the authoring tool's <strong>editing experience</strong>. It helps ensure that content can be created by people with disabilities.</p>
-	<p>Part B (Principles B.1 to B.4) is about the <strong>output</strong> of the tool. It helps ensure that content editors can create accessible content, and are  encouraged to do so.</p>
-	<p><small>Note: this evaluation happens on your computer, no data is sent to us. To save the results on your computer, use <kbd>Ctrl</kbd>/<kbd>⌘ (Command)</kbd> + <kdb>S</kdb>.</small></p>
-</details>
+<p><small>Note: This tool does not automatically save the information that you enter. To save your data in a file locally on your computer, use Windows shortcut keys Ctrl+S or Mac shortcut keys ⌘S to open the Save dialog.</small></p>
```

Comment:
We worked quite a bit on this note in the WCAG-EM Report Tool, including placement, and I hope we can just reuse it unless there is good reason not to.